src/app/machines/views/MachineList/_index.scss

```diff
@@ -112,7 +112,7 @@
   }
 
   .machine-list__divider {
-    margin-top: $spv--x-large;
+    margin-top: $spv--small;
```

Comment:
You should be able to remove this class entirely since it isn't used anywhere else

Reply:
Thanks! You're absolutely right, forgot to remove this once I decided to get rid of it entirely.